README.md

```diff
@@ -1,3 +1,7 @@
-## Microsoft Open Source Code of Conduct
```

Comment:
Should this have its own CoC file that references the .NET one? (That can be another PR, but it should be done).